pom.xml

```diff
@@ -1661,6 +1661,41 @@
                         <groupId>net.revelc.code.formatter</groupId>
                         <artifactId>formatter-maven-plugin</artifactId>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
+                        <version>3.3.0</version>
+                        <dependencies>
+                            <dependency>
+                                <groupId>de.skuzzle.enforcer</groupId>
+                                <artifactId>restrict-imports-enforcer-rule</artifactId>
+                                <version>2.3.1</version>
+                            </dependency>
+                        </dependencies>
+                        <executions>
+                            <execution>
+                                <id>prevent-star-imports</id>
+                                <goals>
+                                    <goal>enforce</goal>
+                                </goals>
+                                <!-- put an explanatory ID here -->
+                                <phase>process-sources</phase>
+                                <configuration>
+                                    <rules>
+                                        <RestrictImports>
+                                            <!-- Define an explanatory reason why these imports are prohibited -->
+                                            <reason>Remove JUnit Framework</reason>
+                                            <!-- Specify a single pattern to be banned -->
+                                            <bannedImport>junit.framework.*</bannedImport>
+                                            <bannedImport>org.junit.*</bannedImport>
+                                            <bannedImport>**.'*'</bannedImport>
```

Comment:
I think this bannedImport entry will prevent any imports containing a `*`.  Is that correct?  Is it worth having this entry in a separate group such that the `reason` is correct for it?

Reply:
Yep that's correct. I'll space this out to another category